tests/OSPSuite.Core.IntegrationTests/ModuleIntegrationTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Linq;
+using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Helpers;
+using OSPSuite.Utility.Container;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Core
+{
+   public abstract class concern_for_ModuleIntegration : ContextForIntegration<IModelConstructor>
```

Comment:
This is the entry point to implement the Use cases that @PavelBal  (hum hum) is working on